db/batch/batch_impl.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	// resetSnapshotIgnoreThreshold is the threshold of snapshot number to ignore reset snapshots
+	resetSnapshotIgnoreThreshold = 10
```

Comment:
`_resetSnapshotIgnoreThreshold`

Reply:
In go, the convention should be to start the name of a private variable with a lowercase letter.

Reply:
in our codebase, lots of private constants name start with _